docker/compose/nginx-flask-mysql.yml

```diff
@@ -1,17 +1,31 @@
 version: '2'
+
 services:
   db:
     image: mysql:5.7
     environment:
-      - MYSQL_ROOT_PASSWORD=XX34!dfd%sd^&
-      - MYSQL_DATABASE=todo_app
-      - MYSQL_USER=web_user
-      - MYSQL_PASSWORD=password
-  web:
-    image: brianmorgan/flask-uwsgi
-    ports:
-      - "8080:5000"
+      MYSQL_ROOT_PASSWORD: XX34!dfd%sd^&
+      MYSQL_DATABASE: todo_app
+      MYSQL_USER: web_user
+      MYSQL_PASSWORD: password
+
+  nginx:
+    image: brianmorgan/nginx:latest
+    depends_on:
+      - web
     environment:
-      - UWSGI_HTTP_PORT=5000
```

Comment:
It's just in reference to the method by which you can structure the `yaml`.  You can do `- UWSGI_HTTP_PORT=5000` or `UWSGI_HTTP_PORT: 5000` which is better picked up by syntax checkers.